src/config/texts.js

```diff
@@ -822,6 +822,9 @@ export const texts = {
       personal: 'Mein Bereich',
       tasks: 'Meine Aufgaben'
     },
+    voucher: {
+      home: 'Treue Club'
```

Comment:
it was "+++ TreueClub +++" in the old app. let's use TreueClub

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/dd7d9179b94aa75f730c71c749f6b75f5995d5a1